internal/core/ebook.go

```diff
@@ -101,9 +105,10 @@ func GenerateEbook(req ProcessRequest) (book model.Bookmark, err error) {
 		return book, errors.Wrap(err, "can't create content.opf")
 	}
 	_, err = contentOpfWriter.Write([]byte(`<?xml version="1.0" encoding="UTF-8"?>
-<package xmlns="http://www.idpf.org/2007/opf" version="2.0" unique-identifier="BookId">
-  <metadata>
+<package xmlns="http://www.idpf.org/2007/opf" version="2.0" unique-identifier="` + ebookIdentifier + `">
+  <metadata xmlns:dc="http://purl.org/dc/elements/1.1/">
```

Comment:
Can we add a custom field in the metadata with the URL from which this ebook has been generated? Is that allowed in the ebook spec?